knowledge-base/shivammathur/cache-extensions/actions-security.yml

```diff
@@ -0,0 +1,27 @@
+name: 'cache-extensions'
+outbound-endpoints:
+  api.github.com:443:
+    port: 443
+    reason: to call GitHub API
+  github.com:443:
+    port: 443
+    reason: to download scripts from GitHub
+  objects.githubusercontent.com:443:
+    port: 443
+    reason: to fetch git object
+  ppa.launchpad.net:80:
+    port: 443
+    reason: to get package files from launchpad PPA
+  artifactcache.actions.githubusercontent.com:443:  
+    port: 443
+    reason: to cache files
+  vstsmms.actions.githubusercontent.com:443:
+    port: 443
+    reason: to cache files
+  pipelines.actions.githubusercontent.com:443:
+    port: 443
+    reason: to restore cached files
+  prtcacprodeus2file7.blob.core.windows.net:443:
```

Comment:
Can you please change this to *.blob.core.windows.net for now? I will need to figure out how to get accurate archive URL at runtime.

Reply:
Can I change to use wildcards elsewhere also, that would reduce the config

Reply:
Done